# Request
string frame_id   # expected frame (optional sanity)

---
# Response
float32 min_x
float32 min_y
float32 min_z
float32 max_x
float32 max_y
float32 max_z
uint32  count     # number of finite points used
